### PR TITLE
Fix #80 create notification screen

### LINKED ISSUE #80: Create NotificationScreen
- add ScrollView

### PR DESCRIPTION
**Pull request by ElisaNgLi**

### CODE REVIEW

File: src/component/Notification/ContentItem.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { scale, moderateScale, verticalScale } from "../../config/scaling";
+import colorConfig from "../../config/colors";
+
+class ContentItem extends Component {
+  render = () => {
```
Comment (hyunjiLeeTech) [inline src/component/Notification/ContentItem.js:7]: Make this component display the list of item (not including status)

File: src/screens/NotificationScreen.js [hunk 2]
```diff
@@ -1,23 +1,72 @@
...
+      name={item.name}
+      vaccineName={item.vaccineName}
+      date={item.date}
+    />
+  );
+
   return (
-    <SafeAreaView style={styles.container}>
-      <Text style={styles.text}>This is notification screen</Text>
+    <SafeAreaView style={screenConfig.container}>
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:53]: Make each status line (upcoming, today and overdue) and make each render funcs filter the items using status (the logic should be changed later. check the status comparing the schedule date and today's device date. We can do it later). You can use ScheduleScreen as a reference.

File: src/screens/NotificationScreen.js [hunk 3]
```diff
@@ -1,23 +1,72 @@
...
+      <View style={styles.container}>
+        <FlatList
+          data={notification}
+          renderItem={renderItem}
+          keyExtractor={(item) => item.id}
+        />
+      </View>
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:60]: You can handle the case when there is not item in certain section. Need to ask a design for this case to Hena

**Review by hyunjiLeeTech — CHANGES_REQUESTED** [review]

File: src/screens/NotificationScreen.js [hunk 4]
```diff
@@ -1,23 +1,141 @@
...
+                name={item.name}
+                vaccineName={item.vaccineName}
+                date={item.date}
+              />
+            );
+          })}
+      </View>
+    );
+  };
+
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:94]: Functions named statusToday, statusUpcoming and statusOverdue are duplicates in terms of the logic of function. Make a one function, pass the status to filter the array

File: src/screens/NotificationScreen.js [hunk 5]
```diff
@@ -1,23 +1,141 @@
...
+    marginBottom: verticalScale(15),
+  },
+  textContainer: {
+    backgroundColor: colorConfig.white,
+    width: scale(349),
+    marginTop: verticalScale(10),
+  },
+  status: {
+    fontFamily: "fredokaOne-regular",
+    fontSize: moderateScale(12),
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:132]: If you want to make the size scaling by the height size of phone, use verticalScale. We don't know the logic of moderateScale so using moderateScale makes the design slightly (but obviously need to fix) different depend on what device we test or use until we've got the logic of moderateScale. Check ContentItem.js too

File: src/component/Notification/ContentItem.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { scale, moderateScale, verticalScale } from "../../config/scaling";
+import colorConfig from "../../config/colors";
```
Comment (hyunjiLeeTech) [inline src/component/Notification/ContentItem.js:4]: Erase the module you don't use on this screen

**Review by hyunjiLeeTech — CHANGES_REQUESTED** [review]

File: src/screens/NotificationScreen.js (hunk 4 above)
Comment (hyunjiLeeTech) [reply src/screens/NotificationScreen.js:94]: When we need to use date to determine the status, we do need 3 separate functions. I delete this. Ignore this review.

**Review by HyoukSunKwon — APPROVED** [review]

File: src/screens/NotificationScreen.js [hunk 7]
```diff
@@ -1,23 +1,172 @@
...
+  blue: {
+    color: colorConfig.blue,
+  },
+  grey: {
+    color: colorConfig.grey,
+  },
+  image: {
+    width: scale(50),
+    height: verticalScale(60),
+  },
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:159]: Make a icon size little bit smiller and add this
resizeMode: "stretch",

File: src/screens/NotificationScreen.js [hunk 8]
```diff
@@ -1,23 +1,172 @@
...
+    flexDirection: "row",
+    marginTop: verticalScale(5),
+    alignItems: "center",
+    alignSelf: "center",
   },
-  text: {
-    textAlign: "center",
+  noStatus: {
+    fontFamily: "fredokaOne-regular",
+    fontSize: verticalScale(12),
```
Comment (hyunjiLeeTech) [inline src/screens/NotificationScreen.js:168]: Make a font size bigger for no status text. When you check Hena's design, you can see the font size is slightly bigger than the section font size

**Review by hyunjiLeeTech — CHANGES_REQUESTED** [review]

**Review by hyunjiLeeTech — APPROVED** [review]

**Pull request merged**